meme-text-service/Dockerfile.dev

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:18.04
+
+RUN apt-get update -y && apt-get -y install tesseract-ocr tesseract-ocr-tam python3 python3-pip
+
+WORKDIR /usr/src/app
+
+COPY requirements.txt ./
+
+RUN pip3 install -r requirements.txt
+
+COPY . .
```

Comment:
line 7 can be replaced with this ?

Reply:
Went through the docker caching layer. Looks like whether python or node will install dependencies only if package.json or req.txt changes.

That is the reason behind line 7